test/test_linalg.py

```diff
@@ -5610,8 +5610,11 @@ def call_torch_fn(*args, **kwargs):
             self.assertEqual([(2, 0, 0), (2, 0)], [A_LU.shape, pivots.shape])
 
     @dtypesIfCUDA(torch.cfloat, torch.cdouble,
-                  *get_all_fp_dtypes(include_half=not CUDA9, include_bfloat16=(CUDA11OrLater and SM53OrLater)))
-    @dtypes(*(set(get_all_dtypes()) - {torch.half, torch.bool}))
+                  *floating_types_and(
+                      *((torch.half, ) if not CUDA9 else ()),
+                      *((torch.bfloat16, ) if not (CUDA11OrLater and SM53OrLater) else ()))
+                  )
+    @dtypes(*(set(all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool)) - {torch.half, torch.bool}))
```

Comment:
Can't this just exclude half and bool without creating a set and removing them?

Reply:
Ohh, yes! I missed it. Thanks:)